### PR TITLE
Add VINF

### PR DESCRIPTION
**Pull request by iphysresearch**
**Variational Inference using Normalizing Flows (VINF)**

PDF: https://pierresegonne.github.io/VINF/
Medium: https://medium.com/swlh/normalizing-flows-are-not-magic-22752d0c924
GitHub: https://github.com/pierresegonne/VINF

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -310,6 +311,14 @@ A list of awesome resources for understanding and applying normalizing flows (NF
 
 <br>
 
+### <img src="assets/tensorflow.svg" alt="TensorFlow" height="20px"> &nbsp;TensorFlow Repos.
```
Comment (janosh) [inline readme.md:314]: Let's remove the trailing dot here.

File: readme.md [hunk 2]
```diff
@@ -310,6 +311,14 @@ A list of awesome resources for understanding and applying normalizing flows (NF
...
+
+1. Jun 12, 2020 - [Variational Inference using Normalizing Flows (VINF)](https://github.com/pierresegonne/VINF) by Pierre Segonne ![GitHub repo stars](https://img.shields.io/github/stars/pierresegonne/VINF)
+
+   >This repository provides a hands-on `tensorflow` implementation of Normalizing Flows as presented in the [paper](https://arxiv.org/pdf/1505.05770.pdf) introducing the concept (D. Rezende & S. Mohamed). The final report of the course, that details all experiments run with this repository can directly be accessed at [PDF](https://pierresegonne.github.io/VINF/).
```
Comment (janosh) [inline readme.md:318]: And let's shorten the description a little to:

```md
   > This repository provides a hands-on TensorFlow implementation of Normalizing Flows as presented in the [paper](https://arxiv.org/pdf/1505.05770.pdf) introducing the concept (D. Rezende & S. Mohamed).
```

**Review by janosh — COMMENTED** [review]
Good stuff! Just 2 nitpicks.

**Review by iphysresearch — COMMENTED** [review]
Nice!

**Comment by janosh** [comment]
If you want me to make those edits, I believe you need to check the box allow edits from maintainers next to this PR. Otherwise you'll need to push another commit here yourself.

**Comment by iphysresearch** [comment]
Sorry for the delay. 
I think I have checked the box to allow you to edit it.

**Comment by janosh** [comment]
Thanks!

**Pull request merged**